src/__tests__/lib/command/util/capabilities-util.test.ts

```diff
@@ -165,32 +174,44 @@ describe('getStandard', () => {
 describe('translateToId', () => {
 	const listFunctionMock = jest.fn<ListDataFunction<CapabilitySummaryWithNamespace>>()
 		.mockResolvedValue(allCapabilitiesWithNamespaces)
+	sortMock.mockReturnValue(sortedCapabilitiesWithNamespaces)
 
 	it('returns input if it is a `CapabilityId`', async () => {
 		const capabilityId = { id: 'capability-id', version: 1 }
 
 		expect(await translateToId('id', capabilityId, listFunctionMock)).toBe(capabilityId)
 
-		expect(apiCapabilitiesListMock).toHaveBeenCalledTimes(0)
+		expect(apiCapabilitiesListMock).not.toHaveBeenCalled()
+		expect(sortMock).not.toHaveBeenCalled()
 	})
 
 	it('returns `CapabilityId` with input as id if input is a string', async () => {
 		const capabilityId = { id: 'capability-id', version: 1 }
 
 		expect(await translateToId('id', capabilityId.id, listFunctionMock)).toStrictEqual(capabilityId)
 
-		expect(apiCapabilitiesListMock).toHaveBeenCalledTimes(0)
+		expect(apiCapabilitiesListMock).not.toHaveBeenCalled()
+		expect(sortMock).not.toHaveBeenCalled()
 	})
 
 	it('returns id when input is integer (index) and is in range', async () => {
-		expect(await translateToId('id', '2', listFunctionMock)).toStrictEqual({ id: 'button', version: 1 })
+		expect(await translateToId('id', '5', listFunctionMock)).toStrictEqual({ id: 'button', version: 1 })
 
-		expect(listFunctionMock).toHaveBeenCalledTimes(1)
-		expect(listFunctionMock).toHaveBeenCalledWith()
+		expect(listFunctionMock).toHaveBeenCalledExactlyOnceWith()
+		expect(sortMock).toHaveBeenCalledExactlyOnceWith(allCapabilitiesWithNamespaces, 'id')
 	})
 
 	it('throws error is out-of-range integer', async () => {
```

Comment:
This existed before this PR, but the description here could be corrected to "throws error when input is out-of-range integer" or "throws error if out-of-range integer".

Reply:
Good catch. I reworded this a bit to reflect that it's using `fatalError` now too and made the text `fails with fatal error for out-of-range integer`.